src/host/ft_host/API_PtyTests.cpp

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "precomp.h"
+
+using WEX::Logging::Log;
+using WEX::TestExecution::TestData;
+using namespace WEX::Common;
+
+class PtyTests
+{
+    BEGIN_TEST_CLASS(PtyTests)
+    END_TEST_CLASS()
+
+    HRESULT SpawnClient(HPCON hPC, wil::unique_process_information& pi)
+    {
+        HRESULT hr = E_FAIL;
+        STARTUPINFOEXW startupInfo{};
+        if (S_OK == (hr = InitializeStartupInfoAttachedToPseudoConsole(&startupInfo, hPC)))
+        {
+            // Launch ping to emit some text back via the pipe

```

Comment:
nit: my comment is bad
```suggestion
            // Launch cmd to emit some text back via the pipe
```